### PR TITLE
Feature/28534 psychometrician report updates

### PR DESCRIPTION
**Pull request by danwilliams**

### CODE REVIEW

File: admin/services/data-access/completed-check.data.service.js [hunk 1]
```diff
@@ -62,14 +62,23 @@ completedCheckDataService.sqlFindOne = async (checkCode) => {
...
-           [mtc_admin].[checkResult] cr ON (chk.id = cr.check_id) JOIN
-           [mtc_admin].[checkStatus] cs ON (chk.checkStatus_id = cs.id)`
+  let select = `
+  SELECT 
+  chk.*,
+  cr.payload,
+  cs.code,
+  cs.description,
+  prr.code restartCode,
+  (SELECT COUNT(id) FROM [mtc_admin].[pupilRestart] pr WHERE pr.check_id = chk.id) restartCount,
```
Comment (jon-shipley) [inline admin/services/data-access/completed-check.data.service.js]: Please remember that restarts can be deleted

File: admin/services/data-access/completed-check.data.service.js [hunk 2]
```diff
@@ -62,14 +62,23 @@ completedCheckDataService.sqlFindOne = async (checkCode) => {
...
+  ac.code attendanceCode
+  FROM [mtc_admin].[check] chk
+      LEFT JOIN [mtc_admin].[checkResult] cr ON (chk.id = cr.check_id)
+      LEFT JOIN [mtc_admin].[pupilRestart] pr ON (pr.check_id = chk.id AND isDeleted = 0)
+      LEFT JOIN [mtc_admin].[pupilRestartReason] prr ON (prr.id = pr.pupilRestartReason_id)
+      LEFT JOIN [mtc_admin].[pupilAttendance] pa ON (pa.pupil_id = chk.pupil_id AND pa.isDeleted = 0)
+      LEFT JOIN [mtc_admin].[attendanceCode] ac ON (ac.id = pa.attendanceCode_id)
+      JOIN [mtc_admin].[checkStatus] cs ON (chk.checkStatus_id = cs.id)
```
Comment (GeorgeChatzigiannis) [inline admin/services/data-access/completed-check.data.service.js]: should you or not take into account attendance codes LEFT or INCORRECT REGISTRATION for this?

File: admin/services/data-access/completed-check.data.service.js (hunk 1 above)
Comment (danwilliams) [reply admin/services/data-access/completed-check.data.service.js]: Good spot!

File: admin/services/data-access/completed-check.data.service.js (hunk 2 above)
Comment (danwilliams) [reply admin/services/data-access/completed-check.data.service.js]: Thanks - based on the conversation we had about this, all is good.

**Pull request merged**